### ISSUE TITLE
about the center_coordinate and the radius

### ISSUE THREAD

**Issue by centosrhel** [showlab/DragAnything#10]
In the paper, the center_coordinate is set to be the center of the mask's incircle, but in the demo.py file, the center_coordinate is manually written in the demo.json file. 
1. What does this discrepancy mean?
2. There's also one question about the radius, which is constrained to be no larger than 50 by `side = min(radius, 50)`. Why do we need this constraint?

**Comment by weijiawu** [comment]
1. There is no discrepancy there. In the paper, the description of the center of the mask's incircle refer to the process of Ground Truth Label Generation, which is only used during the training process. During inference, the user directly drags a trajectory and uses the trajectory points as the center coordinate.     2. "50" is just a hyperparameter; you can remove this restriction or adjust it.

**Comment by centosrhel** [comment]
got it, thanks for your reply.
